public/static/locales/en/app_editor_help.json

```diff
@@ -0,0 +1,43 @@
+    "ReferenceSequence": "<p>Allow you to add drop-down lists of a <b>Reference Sequence</b> from which the user can make one selection.</p><p>The Discovery Environment has reference sequences uploaded and available for use.</p><p>To add a reference species that is not currently supported, please email the CyVerse support team at support@cyverse.org.</p>",
+    "Regex": "Indicates that the text the user enters must match the expression you define.",
+    "RepeatOptionFlag": "Selecting this option allows the \"Argument option\" defined for this parameter to be included on the command line before each filename selected by the user.",
+    "Section": "<p>An app can contain multiple <b>sections</b> that appear as collapsible panels to help organize the app's content.</p><p>For example, you may have one section labeled \"Inputs\" where users select their input data, as well as an \"Options\" section where users select from the various options you have created for your app.</p>",
+    "Text": "Will display a text box the user may enter information into. You can also set informative or descriptive default \"background\" text to display in the text box.",
+    "TextSelection": "Adds a drop-down list from which the user can make one selection from the options you define.",
+    "TextSelectionEditor": "<p>This section is where you will build your list.</p><p>Click \"Add\", then edit the text in the \"Display\" column to what you would like the user to actually see. Assign an argument to that Display item in the \"Argument\" column, for example \"-n\", then enter that list item's value in the \"Value\" column, such as \"5\".</p><p>When the user clicks that display text in the drop-down, it will be associated with the argument and its value you assigned and will be passed to the command line.</p><p>To select which item you wish to automatically display, select an item from the \"Default value\" list.</p>",
+    "ToolUsed": "A <b>tool</b> is the software program (binary or script) that is installed in a Docker image and integrated into the backend of the Discovery Environment (DE) for use in DE apps.<br/>An <b>app</b> is the <i>interface</i> of a tool that has been configured and made available for use in the DE.<br/>The same <i>tool</i> can be used in more than one <i>app</i>, but an app is created by different integrators with different intentions for that same tool.",
+    "validationRules": "<p><b>Validation rules</b> are the rules you define describing what value the user may enter that will successfully execute the app. If a user enters a value that does not follow the rules you set, the app will display a warning with explanation and not launch the app.</p>"
+}
```

Comment:
Not sure either. It makes sense to me that the rule is validating which values will allow the app to successfully run (for the analysis to complete successfully); but `to successfully launch the app` makes sense as well, since a validation error will prevent an app from launching from the UI.

I think I mentioned this before, but most of this help text was imported from the DE v1 repo; though I did tweak some text that needed obvious changes, like removing references to tool-tips.